tests/unit/EmojiTest.js

```diff
@@ -26,38 +27,38 @@ describe('EmojiTest', () => {
 
-        // GIVEN an emoji that different cross-platform variations WHEN we check if it's a single emoji THEN it should return true
-        expect(EmojiUtils.isSingleEmoji('🔫️')).toBe(true);
-        expect(EmojiUtils.isSingleEmoji('🛍')).toBe(true);
-        expect(EmojiUtils.isSingleEmoji('🕍')).toBe(true);
+        // GIVEN an emoji that different cross - platform variations WHEN we check if it's a single emoji THEN it should return true
+        expect(EmojiUtils.containsOnlyEmojis('🔫️')).toBe(true);
+        expect(EmojiUtils.containsOnlyEmojis('🛍')).toBe(true);
+        expect(EmojiUtils.containsOnlyEmojis('🕍')).toBe(true);
 
         // GIVEN an emoji that is symbol/numerical WHEN we check if it's a single emoji THEN it should return true
-        expect(EmojiUtils.isSingleEmoji('*️⃣')).toBe(true);
-        expect(EmojiUtils.isSingleEmoji('1️⃣️')).toBe(true);
+        expect(EmojiUtils.containsOnlyEmojis('*️⃣')).toBe(true);
+        expect(EmojiUtils.containsOnlyEmojis('1️⃣')).toBe(true);
 
         // GIVEN an emoji that has text-variant WHEN we check if it's a single emoji THEN it should return true
-        expect(EmojiUtils.isSingleEmoji('❤️')).toBe(true);
-        expect(EmojiUtils.isSingleEmoji('⁉️')).toBe(true);
-        expect(EmojiUtils.isSingleEmoji('✳️')).toBe(true);
-        expect(EmojiUtils.isSingleEmoji('☠️')).toBe(true);
+        expect(EmojiUtils.containsOnlyEmojis('❤️')).toBe(true);
+        expect(EmojiUtils.containsOnlyEmojis('⁉️')).toBe(true);
+        expect(EmojiUtils.containsOnlyEmojis('✳️')).toBe(true);
+        expect(EmojiUtils.containsOnlyEmojis('☠️')).toBe(true);
 
         // GIVEN an emoji that has skin tone attached WHEN we check if it's a single emoji THEN it should return true
-        expect(EmojiUtils.isSingleEmoji('👶🏽')).toBe(true);
-        expect(EmojiUtils.isSingleEmoji('👩🏾')).toBe(true);
-        expect(EmojiUtils.isSingleEmoji('👊🏾')).toBe(true);
+        expect(EmojiUtils.containsOnlyEmojis('👶🏽')).toBe(true);
+        expect(EmojiUtils.containsOnlyEmojis('👩🏾')).toBe(true);
+        expect(EmojiUtils.containsOnlyEmojis('👊🏾')).toBe(true);
 
         // GIVEN an emoji that is composite(family) with 4+ unicode pairs WHEN we check if it's a single emoji THEN it should return true
-        expect(EmojiUtils.isSingleEmoji('👨‍👩‍👦️')).toBe(true);
-        expect(EmojiUtils.isSingleEmoji('👩‍👩‍👧‍👦️')).toBe(true);
+        expect(EmojiUtils.containsOnlyEmojis('👨‍👩‍👦️')).toBe(true);
+        expect(EmojiUtils.containsOnlyEmojis('👩‍👩‍👧‍👦️')).toBe(true);
 
-        // GIVEN an emoji that has a length of 2 (flags) WHEN we check if it's a single emoji THEN it should return true
-        expect(EmojiUtils.isSingleEmoji('🇺🇲')).toBe(true);
-        expect(EmojiUtils.isSingleEmoji('🇮🇳')).toBe(true);
-        expect(EmojiUtils.isSingleEmoji('🇺🇦️')).toBe(true);
+        // // GIVEN an emoji that has a length of 2 (flags) WHEN we check if it's a single emoji THEN it should return true
```

Comment:
```suggestion
        // GIVEN an emoji that has a length of 2 (flags) WHEN we check if it's a single emoji THEN it should return true
```

Reply:
A lot of the comments still reference "a single emoji". Can you please update the comments to speak instead about only containing emojis?

Also, can you please add tests that confirm:
- Multiple emojis, with no text (and any amount of whitespace), returns true
- single emojis, with some text, returns false
- multiple emojis, with some text, returns false

Reply:
@tgolen I've added tests and fixed your comments.